knowledge_graph/classifier/bert_based.py

```diff
@@ -22,6 +32,27 @@
 from knowledge_graph.labelled_passage import LabelledPassage
 from knowledge_graph.span import Span
 
+logger = logging.getLogger(__name__)
+if not logger.handlers:
+    logger.setLevel(logging.INFO)
+    handler = RichHandler(rich_tracebacks=True)
+    handler.setFormatter(logging.Formatter("%(message)s"))
+    logger.addHandler(handler)
+    logger.propagate = False
```

Comment:
what is this custom logging logic for? it'd be good to see a comment to explain why it's needed